app/src/app.py

```diff
@@ -129,8 +131,10 @@ async def check_email_results(request: Request, recipient_username: str) -> Resp
         error = translate("Invalid or no e-mail domain in the message From header", Language(Config.UI.LANGUAGE))
     else:
         try:
+            client_ip = request.client.host if request.client else None
+            client_user_agent = request.headers.get("user-agent", None)
+
             result = scan_and_log(
```

Comment:
what is the reason scanning of the e-mail domain and DKIM signature is performed synchronously and scanning the domain is async?